playwright/test/view-items.test.ts

```diff
@@ -56,357 +27,250 @@ const test = base.extend({
-    await itemWithSearchAndStructures(context, page);
-    await page.waitForSelector(mainViewer);
-    await scrollToBottom(mainViewer, page);
-
-    // In this test, we're loading an item with 68 pages, scrolling to the
-    // bottom, then looking for the "68/68" text on the page.
-    //
-    // This text is hidden whenever the window is being scrolled, zoomed,
-    // or resized, because that might affect what the "current" page is.
-    //
-    // We've had issues with this test being flaky, because we don't wait
-    // long enough after we finish scrolling to look for this "68/68" --
-    // tossing in this wait seems to fix that.
-    await safeWaitForNavigation(page);
+test('(10) | The volumes should be browsable', async ({ page, context }) => {
+  await multiVolumeItem(context, page);
+  if (isMobile(page)) {
+    await page.getByRole('button', { name: 'Show info' }).click();
+  }
+  await page.getByRole('button', { name: 'Volumes' }).click();
+  await expect(page.getByRole('link', { name: 'Copy 3' })).toBeVisible();
+});
 
-    await page.waitForSelector(
-      `css=[data-test-id=active-index] >> text="68"`,
+test('(11) | The multi-volume label should be appropriate', async ({
+  page,
+  context,
+}) => {
+  await multiVolumeItem(context, page);
+  if (isMobile(page)) {
+    await page.getByRole('button', { name: 'Show info' }).click();
+  }
+  expect(await page.getByText('Copy 1'));
+  await page.getByRole('button', { name: 'Volumes' }).click();
+  await page.getByRole('link', { name: 'Copy 3' }).click();
+  expect(await page.getByText('Copy 3').count());
+});
 
-      // The "68/68" label isn't visible on small screens, but the element
-      // will still be on the page.
-      { state: isMobile(page) ? 'attached' : 'visible' }
-    );
-  });
+test('(12) | The structured parts should be browseable', async ({
+  page,
+  context,
+}) => {
+  await multiVolumeItem(context, page);
+  if (isMobile(page)) {
+    await page.getByRole('button', { name: 'Show info' }).click();
+  }
+  expect(await page.getByTestId('active-index').textContent()).toEqual('1');
+  await page.getByRole('button', { name: 'Contents' }).click();
+  await page.getByRole('link', { name: 'Title Page' }).click();
+  if (!isMobile(page)) {
+    // we don't display this info on mobile as there is not enough room
+    await expect(await page.getByText('9/559')).toBeVisible();
+  }
 });
 
-test.describe("Scenario 9: A user wants to be able to search inside an item's text", () => {
-  test('the item should be searchable', async ({ page, context }) => {
-    await itemWithSearchAndStructures(context, page);
-    await safeWaitForNavigation(page);
-    if (isMobile(page)) {
-      await page.click('text="Show info"');
+test('(13) | The main viewer can be scrolled', async ({ page, context }) => {
+  await itemWithSearchAndStructures(context, page);
+  await expect(page.locator('[data-testid="main-viewer"] > div')).toBeVisible();
```

Comment:
If you want, I _think_ you could do something like
`page.getByTestId('main-viewer').locator('> div')` but would need to be tested.